Dockerfiles/cluster-agent/Dockerfile

```diff
@@ -1,11 +1,11 @@
 ########################################
 # Preparation stage: layout and chmods #
 ########################################
-
-FROM ubuntu:22.04 as builder
+ARG BASE_IMAGE_UBUNTU_VERSION=23.04
+FROM ubuntu:$BASE_IMAGE_UBUNTU_VERSION as builder
 ARG TARGETARCH
-LABEL baseimage.os "ubuntu jammy LTS"
-LABEL baseimage.name "ubuntu:22.04"
+LABEL baseimage.os "ubuntu lunar"
```

Comment:
Do we want to add this string as an ARG as well?

Reply:
yes, 
the idea in futur PR is to use the `release.json` to store the base info image, and provide this info to the `invoke build` command